src/cls/rgw/cls_rgw.cc

```diff
@@ -2215,9 +2217,18 @@ int rgw_dir_suggest_changes(cls_method_context_t hctx,
       }
     }
 
-    CLS_LOG(20, "cur_disk.pending_map.empty()=%d op=%d cur_disk.exists=%d cur_change.pending_map.size()=%d cur_change.exists=%d",
+    CLS_LOG(20, "cur_disk.pending_map.empty()=%d op=%d cur_disk.exists=%d "
```

Comment:
it's amazing this step was missed before (or maybe got removed?); it seems clearly part of the design

Reply:
I think the dir_suggest code path might not have been tested much

Reply:
I'm guessing this will merge before my instrumentation PR, which it should. Then I can convert this to the instrumented version.